desktop/apps/the_armory_show/components/ArmoryShowFairWeekPage.js

```diff
@@ -59,7 +60,40 @@ const theme = {
   }
 }
 
-export default ({ introduction, fair_coverage, event, prepare_for_fairs }) =>
+const Cta = styled.div`
+  cursor: pointer;
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  height: 100px;
+  width: 100%;
+  background-color: #fff;
+  z-index: 999;
+  border-top: 2px solid #6e1fff;
+`
+
+const CtaImage = styled.img`
+  box-flex: 1;
```

Comment:
Never saw this attr before, but it looks like support for it is pretty low: https://developer.mozilla.org/en-US/docs/Web/CSS/box-flex

Is it similar to flex-grow?